cronjob_scripts/trigger-package-build.py

```diff
@@ -80,7 +80,7 @@ def trigger_for_arch(target_arch: str):
             features = "vendored-openssl"
         else:
             features = ",".join(
-                feat for feat in version["features"].keys() if "vendored" in feat
+                feat for feat in version["features"].keys() if "vendored" in feat or bundled in feat
```

Comment:
```suggestion
                feat for feat in version["features"].keys() if "vendored" in feat or "bundled" in feat
```

I feel like Ruff would have caught this.

Reply:
Oops thanks.

I think maybe we want mypy/ruff/ruff-format in CI?